gamestonk_terminal/fundamental_analysis/av_view.py

```diff
@@ -517,3 +517,176 @@ def clean_fundamentals_df(df_fa: pd.DataFrame, num: int) -> pd.DataFrame:
     df_fa.columns.name = "Fiscal Date Ending"
 
     return df_fa
+
+
+def mscore(other_args: List[str], ticker: str):
+    """Mscore for given ticker
+
+    Parameters
+    ----------
+    other_args : List[str]
+        argparse other args
+    ticker : str
+        Fundamental analysis ticker symbol
+    """
+    parser = argparse.ArgumentParser(
+        add_help=False,
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        prog="cash",
```

Comment:
the `prog` value needs to represent the command used to call this. I.e. in this case should be `fraud`